common/testData/personNameTest/af.txt

```diff
@@ -64,33 +64,51 @@ name ; locale; af_AQ
 
 expectedResult; Piet
 
-parameters; none; long; addressing; formal
-parameters; none; long; addressing; informal
-parameters; none; long; referring; formal
-parameters; none; long; referring; informal
-parameters; none; medium; addressing; formal
-parameters; none; medium; addressing; informal
-parameters; none; medium; referring; formal
-parameters; none; medium; referring; informal
-parameters; none; short; addressing; formal
-parameters; none; short; addressing; informal
-parameters; none; short; referring; formal
-parameters; none; short; referring; informal
+parameters; givenFirst; long; addressing; formal
+parameters; givenFirst; long; addressing; informal
+parameters; givenFirst; long; referring; formal
+parameters; givenFirst; long; referring; informal
+parameters; givenFirst; medium; addressing; formal
+parameters; givenFirst; medium; addressing; informal
+parameters; givenFirst; medium; referring; formal
+parameters; givenFirst; medium; referring; informal
+parameters; givenFirst; short; addressing; formal
+parameters; givenFirst; short; addressing; informal
+parameters; givenFirst; short; referring; formal
+parameters; givenFirst; short; referring; informal
```

Comment:
As you mention, this'll require a change on the ICU side, but I don't think it'll be a big deal.  My only real concern about that is that the exhaustive ICU tests will fail until I can get in there and fix it to deal with this (not to mention any other issues this uncovers), and I've already gotten no end of flak for that test failing over the past few weeks.  I'd like to find a way to coordinate our changes so that we don't have that period of failing tests.

Any thoughts on how best to accomplish that?  One option might be to leave the test data files on the ICU side alone until I have the corresponding code changes ready to go.  Then I can include the updated data files as part of my PR.  Is there a ay to make that happen, or is the whole process automated now?

Reply:
Good question. This PR should fix the problems with the discontinuity between the locale PNF data and the PNF test file. We will not be able to check in data that affects the test without regenerating the testData. So that will prevent future integration problems with ICU.

Reply:
I guess what I'm worried about here is if we still have any algorithmic differences between CLDR and ICU.  If we do, the next CLDR integration will cause a crisis on the ICU side when this test starts failing.  And come to think of it, we _know_ we're going to have an algorithmic difference because you're using new syntax in the test file that my unit test doesn't yet know about.

Maybe it's not a big crisis if we keep this test as part of the exhaustive tests, but Frank wants to move it to the regular test suite (and usually yells at me if I break the exhaustive tests anyway).  Maybe we just have to live with that, but I'm not looking forward to it.

The other choice would be for me to try to get ahead of things by fixing my code _before_ the new data files land in the ICU build.  Maybe we should figure out a way to do that?